### DISCUSSION TITLE
Export area

### DISCUSSION THREAD

**Discussion by mariekfmn** [biigle/core#979] [Q&A]
Hi everyone, 
is there a way to set the export area so that it covers a certain image area (m2) in each image? At the moment, I have only found the possibility to draw a frame that stays the same for all images regardless of the size of the images (which differs in my case.) 
My problem is that I have already started with my image annotation and unfortunately found out afterward that I should have adjusted my image size. So I am now looking for a way to avoid having to discard my previous work and simply try to find a way to only export the annotations in a certain area in the image. 
Help is very appreciated :)

**Comment by mzur** [comment]
This is not possible with the export area, sorry. If you know the area of each image, you could create rectangle annotations with a fixed (m²) size via the API and a custom script. Here is [an example](https://github.com/biigle/community-resources/tree/master/regular-sampling-grid) for a similar script that creates a regular grid. This could give you a visual cue of the area to annotate.

This would still not work like the export area, though, as it does not allow you to automatically exclude annotations in a report. You can either delete annotation outside the area manually or filter the annotations after exporting them from BIIGLE.
